vm/actor/tests/common/mod.rs

```diff
@@ -431,7 +459,16 @@ impl<BS: BlockStore> Runtime<BS> for MockRuntime<'_, BS> {
         }
         self.balance -= value;
 
-        return Ok(expected_msg.send_return);
+        match expected_msg.exit_code {
+            ExitCode::Ok => return Ok(expected_msg.send_return),
+            x => {
+                return Err(ActorError {
+                    fatal: false,
+                    exit_code: x,
+                    msg: "Expected message Fail".to_string(),
+                });
+            }
+        }
```

Comment:
This seems wrong, why are you doing this?

Reply:
In the go version if the command was expected to fail they return the failed exit code. I assumed the best way to mimic that was to return a error. Before we would always return Ok(expected_msg.send_return) even if the expected message was supposed to fail

Reply:
Oh okay I didn't realize that error was not handled before. @ec2 can you verify this, since you set up this function? Seems weird to attach an arbitrary message to the error, but I guess maybe that's necessary to port tests over from their framework

Reply:
Yeah, I think i messed that one up there, or their mock rt changed a bit. Not sure which one, but @nannick 's changes look correct